datadog_checks_dev/datadog_checks/dev/conditions.py

```diff
@@ -45,7 +45,11 @@ def __call__(self):
 
             time.sleep(self.wait)
         else:
-            raise RetryError('Result: {}\n' 'Error: {}'.format(repr(last_result), last_error))
+            raise RetryError(
```

Comment:
Let's do this in a separate PR

Reply:
Sure, moved to this PR: https://github.com/DataDog/integrations-core/pull/4619